ComposeGallery/src/androidTest/java/com/microsoft/device/display/samples/composegallery/PaneSynchronizationTest.kt

```diff
@@ -49,7 +50,7 @@ class PaneSynchronizationTest {
      */
     @ExperimentalTestApi
     @Test
-    fun app_testListItemClickUpdatesDetailPane() {
+    fun app_verticalFold_testListItemClickUpdatesDetailPane() {
         composeTestRule.setContent {
             ComposeGalleryTheme {
                 ComposeGalleryApp(composeTestRule.activity.rememberWindowState())
```

Comment:
Can we use `WindowState()` here, instead of `rememberWindowState()`?

Reply:
Yes, good catch!